lib/fizzy/instantiate.hpp

```diff
@@ -21,8 +21,31 @@ namespace fizzy
 struct ExecutionResult;
 struct Instance;
 
+class ThreadContext
```

Comment:
Can the declaration be moved to `execute.hpp`, as `ExecutionResult` ?

Reply:
Maybe.